src/utils/promptRunner.ts

```diff
@@ -0,0 +1,107 @@
+import { spawn } from "child_process";
+import { window } from "vscode";
+const output = window.createOutputChannel("Docker Labs: AI Tools");
+
+export const getRunArgs = (promptRef: string, projectDir: string, username: string, platform: string, pat: string, render = false) => {
+    const isLocal = promptRef.startsWith('local://');
+    let promptArgs: string[] = ["--prompts", promptRef];
+    let mountArgs: string[] = [];
+
+    if (isLocal) {
+        const localPromptPath = promptRef.replace('local://', '');
+        const pathSeparator = platform === 'win32' ? '\\' : '/';
+        promptRef = localPromptPath.split(pathSeparator).pop() || 'unknown-local-prompt';
+        promptArgs = ["--prompts-dir", `/app/${promptRef}`];
+        mountArgs = ["--mount", `type=bind,source=${localPromptPath},target=/app/${promptRef}`];
```

Comment:
We now also support --prompts-file so that you can point at just one file that contains both prompt sections and metadata for setting the function calls.  In this case, you need to mount the parent dir of the prompts file.  I think it should be straight forward to check whether the promptRef is a file or directory; however, if it's a file, you still need to mount the parent dir and then point at the file using `--prompts-file` so it's a little bit more complicated here.